oc-chef-pedant/spec/api/controls_spec.rb

```diff
@@ -12,419 +12,432 @@
 
 describe "Controls API Endpoint", :controls do
 
-  describe "v0.1.0" do
-    include Pedant::RSpec::NodeUtil
-
-    # The reason for this is that we need to verify
-    # that the principal endpoint functions properly in OPC, as
-    # currently the only consumer of this endpoint is chef-client-audit which
-    # absolutely requires the OPC control endpoint to work.
-    let(:input) {
-      {
-        :message_type => "control_groups",
-        :message_version => "0.1.0",
-        :organization_name => "ponyville",
-        :chef_server_fqdn => "api.opscode.piab",
-        :recorded_at => "2014-11-07T14:53:58Z",
-        :remote_hostname => "33.33.33.1",
-        :request_id => "g3IAA2QAEGVyY2hlZkAxMjcuMC4wLjEDAABM1wAAAAAAAAAA",
-        :node_name => "client.opscode.piab",
-        :id => "a4247480-bb92-42a5-8116-3729dcc486eb",
-        :run_id => "76296628-7de0-4509-adaa-9a525409142d",
-        :control_groups => [
-          {
-            :name => "mysql audit",
-            :status => "failure",
-            :number_success => 1,
-            :number_failed => 1,
-            :id => "59186fdc-2d0a-4653-af83-bbe79302fcaf",
-            :controls => [
-              {
-                :name => "should be installed",
-                :status => "success",
-                :details => "null",
-                :resource_type => "Package",
-                :resource_name => "mysql",
-                :context => [],
-                :sequence_number => 1
-              },
-              {
-                :name => "should have mysql service",
-                :status => "failure",
-                :details => "expected Service \"mysql\" to be enabled",
-                :resource_type => "null",
-                :resource_name => "null",
-                :context => [],
-                :sequence_number => 2
-              }
-            ]
-          }
-        ]
+  if Pedant::Config.actions_enabled
```

Comment:
The closest we have is LDAP testing, where config variable drives enabling various LDAP auth-related tests. Other than that, we have done this for darklaunch-flagged features in the past (though all of those have been cleaned up). 

As with LDAP and previous darklaunch, there's really no way that our current test setup can handle it. 


Alternative: I'd  given thought to having the test toggle the config flag via rpc- that'd let us  just test both.  I decided against it because time but we can revisit it.  Having that ability built into pedant would actually open up some interesting additional testing options.